### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJEclipse_Deploy to Azure_Keyboard: As soon as the focus lands on "Create New WebApp" list item, it gets selected

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#3376]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier!

**NOTE: Same is the case with the WebApp combo-box present in all other screens**

**Environment Details:**
Download and Install IntelliJ IDEA from https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable
IntelliJ IDEA 2019.1.3
Build #IU-191.7479.19

**OS Details: ​**
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS 3.2.1

**Repro Steps:**
1) Open IntelliJ IDEA application
2) Press 'Ctrl+Alt+S' to open application settings/preferences popup
3) Tab till 'Plugins' tab and press enter
4) Search for 'Azure Toolkit for IntelliJ in the Marketplace' and press enter.
5) Tab till 'Install' button and press enter
6) Restart IntelliJ IDEA application, and tab till Tools-->Azure-->Azure Sign in
7) Sign in with any Azure subscription
8) Tab till File menu and create new maven-archetype-webapp project
9) Under Project Explorer view, open and edit the file src/main/webapp/index.jsp
10) Under Project Explorer view, right-click your project, expand Azure, then click Deploy to Azure
11) Tab till the "WebApp" combo-box and expand it
12) Tab till the "Create New WebApp" list items and check if it gets selected automatically as soon as it receives the focus

**Actual Result:**
As soon as the focus lands on "Create New WebApp" list item, it gets selected

**Expected Result:**
"Create New WebApp" list item should be selected only when the user presses enter key

**User Impact:**
Users will find it difficult to access the "WebApp" combo-box

**MAS Reference:**
[MAS 3.2.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7Ba4bbd37d-ee11-485e-b5c8-0539f0ac94b2%7D&cid=cb415926-9190-4e6f-a6f3-1f669d5df313)

[Bug_3376_Deploy To Azure_WebApp_OnFocus.pptx](https://github.com/microsoft/azure-tools-for-java/files/3460215/Bug_3376_Deploy.To.Azure_WebApp_OnFocus.pptx)

**Comment by andxu** [comment]
Post the snapshot to help better understanding of this issue
![image](https://user-images.githubusercontent.com/17075863/64404735-727a9080-d0af-11e9-8cc2-579e785f13e0.png)

**Comment by Flanker32** [comment]
@soumyatippapuram  This issue has been fixed in #3541, you may find the build [here](https://vscjavaci.cloudapp.net/job/Azure_Toolkits_for_Java/job/AzureToolsManager_develop/1250/Azure/), please help us verify it, thanks a lot!

**Comment by soumyatippapuram** [comment]
Verified this issue in the above mentioned build. This issue is fixed. Hence, closing this. Added attachment for reference.
[Bug_3376_WebApp_OnFocus_Fixed.pptx](https://github.com/microsoft/azure-tools-for-java/files/3620821/Bug_3376_WebApp_OnFocus_Fixed.pptx)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749771
